contributors/design-proposals/cpu-manager.md

```diff
@@ -158,6 +163,26 @@ node. Once allocated at pod admission time, an exclusive CPU remains
 assigned to a single container for the lifetime of the pod (until it
 becomes terminal.)
 
+##### Configuration
+
+Operators can set the number of CPUs that pods may run on through a new
+Kubelet configuration setting `--cpu-manager-static-num-cpus`, which
```

Comment:
Linking parallel discussion in @flyingcougar 's PR (https://github.com/ConnorDoyle/kubernetes/pull/5).

Will update this to go back to using Allocatable for this value, agree it makes more sense now. Sorry for the confusion.